docs/faq-security.md

```diff
@@ -62,23 +65,19 @@ Worklytics that is otherwise not exposed to the internet.
 
 ## Can I deploy Psoxy instances in a VPC?
 
-As of November 2023, that is not directly supported by the Worklytics-provided Terraform modules. If
-you have a use-case that you believe requires a VPC, please let us know.
+VPC support is available as a *beta* feature as of February 2024.
```

Comment:
Reason for this date?

Reply:
just informational ... we used to tell people it wasn't supported, but now we've done it.